src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.wasm.xml

```diff
@@ -0,0 +1,142 @@
+<linker>
+  <assembly fullname="*">
+    <!-- System -->
+    <type fullname="System.CLSCompliantAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+    <type fullname="System.ObsoleteAttribute">
+      <attribute internal="RemoveAttributeInstances" />
+    </type>
+    <type fullname="System.ParamArrayAttribute">
```

Comment:
`ParamArrayAttribute` is considered by reflection for method binding. This will affect how reflection Invoke behaves.

Reply:
Didn't think about that, removed